gradle/libs.versions.toml

```diff
@@ -100,6 +100,7 @@ immutables-builder = { module = "org.immutables:builder", version.ref = "immutab
 immutables-value-annotations = { module = "org.immutables:value-annotations", version.ref = "immutables" }
 immutables-value-processor = { module = "org.immutables:value-processor", version.ref = "immutables" }
 jackson-bom = { module = "com.fasterxml.jackson:jackson-bom", version = "2.16.0" }
+jackson-smile = { module = "com.fasterxml.jackson.dataformat:jackson-dataformat-smile", version = "2.16.0" }
```

Comment:
Why not import the bom and just mention the dependency?

Reply:
It's not part of the bom :-(

Reply:
Hmm actually it is there, but when I tried it didn't work. I'll dig further.

Reply:
It's working now.